shared_test_logic/basic_batch_logic.dart

```diff
@@ -0,0 +1,194 @@
+import 'package:kiss_repository/kiss_repository.dart';
+
+import 'data/product_model.dart';
```

Comment:
i wonder if we should always name the test models with test_ just to make sure its clear?

Reply:
ok, will rename!